docs/content/docs/dev/table/jdbcDriver.md

```diff
@@ -228,8 +252,8 @@ public class Sample {
 }
 ```
 
-### Use with Others
+### Other languages
 
-In addition to java, Flink JDBC driver can be used by any JVM language such as scala, kotlin and ect, you can add the dependency of Flink JDBC driver in your project and use it directly.
+In addition to Java, the Flink JDBC driver can be used by any JVM language such as Scala, Kotlin etc. Add the dependency of Flink JDBC driver in your project and use it directly.
 
-Most applications may use data access frameworks to access data, for example, JOOQ, MyBatis and Spring Data. You can config Flink JDBC driver in them to perform Flink queries on an exist Flink cluster, just like a regular database.
+Most applications may use data access frameworks to access data, for example, JOOQ, MyBatis and Spring Data. You can configure the Flink JDBC driver in them to perform Flink queries on an existing Flink cluster, just like a regular database.
```

Comment:
This paragraph is still difficult to understand. Let's clean it up; maybe something like the suggestion below. I also think this content should be moved up so that it becomes an introductory paragraph in the Use with Application section, instead of being buried here.

```suggestion
Many applications access data in SQL databases, either directly, or through frameworks like JOOQ, MyBatis, and Spring Data. You can configure these applications and frameworks to use the Flink JDBC driver so that they perform SQL queries on a Flink cluster instead of a regular database.
```